Dockerfile.wasi-builder

```diff
@@ -1,6 +1,8 @@
 FROM ghcr.io/vmware-labs/wasmlabs/wasm-base:latest
 ARG WASI_SDK_VERSION
 ARG WABT_VERSION=1.0.32
+ARG WASI_VFS_VERSION=0.2.0
```

Comment:
When I was researching about Wasi-vfs, I found that v0.2.0 doesn't include support for bulk memory. This was added to wizer [in this commit](https://github.com/bytecodealliance/wizer/commit/65e345fb2bd5b5177d4be55af747edc28035c35c), and Wasi-vfs pointed to this version [in their source code](https://github.com/kateinoigakukun/wasi-vfs/commit/bab98802249ead4b2cfdc93d74d156fc099c7412).

However, that happened after the latest release on August ([v0.2.0](https://github.com/kateinoigakukun/wasi-vfs/releases/tag/v0.2.0)). I think for Ruby, v0.2.0 works properly but our Python module requires those changes. We may need to contact the wasi-vfs owner to release a new version or compile it by ourselves meanwhile 🤔

Reply:
Thanks, updated :)